lib/modules/job/widgets/jobCard.widget.dart

```diff
@@ -0,0 +1,106 @@
+
+class JobCard extends State<JobCardWidget> {
+  static const String fillerText = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. '
+      'Fusce rhoncus urna ut ex rutrum convallis. Suspendisse potenti. Vivamus nisl ipsum, '
+      'ornare non sodales nec, tincidunt eu eros. Nam sollicitudin ante ligula. ';
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
+      child: Card(
+        child: Column(
+          mainAxisSize: MainAxisSize.min,
+          children: <Widget>[
+            const ListTile(
+              title: Text('Job Title'),
+            ),
+            const ListTile(
+              title: Text(
+                fillerText,
+                textAlign: TextAlign.left,
+              )
+            ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+              mainAxisSize: MainAxisSize.max,
+              children: <Widget>[
+                Expanded(child:
+                  InkWell(
+                    splashColor: Colors.blue.withAlpha(30),
+                    onTap: () {
+                      print('Card tapped.');
+                    },
+                    child: Card(
+                      child: ListTile(
+                        leading: Icon(Icons.album),
+                        title: Text("test"),
+                        subtitle: Text("subtext"),
+                      ),
+                    ),
+                  )
+                ),
+                Expanded(
+                  child: InkWell(
+                    onTap: (){
+                      print('Another card tapped.');
+                    },
+                    child: Card(
+                      child: ListTile(
+                        leading: Icon(Icons.album),
+                        title: Text("test"),
+                        subtitle: Text("another text"),
+                      ),
+                    ),
+                  ),
+
+                )
+
+                ],
+              ),
+            Row(
+              mainAxisAlignment: MainAxisAlignment.end,
+              children: [
+                Text(
+                  "Posted by"
+                ),
+                SizedBox(width: 12,),
+                Card(
+                  child: Row(
+                    mainAxisSize: MainAxisSize.min,
+                    children: [
+                      Icon(Icons.album),
+                      TextButton(
+                          onPressed: (){
+                            print("Poster name clicked");
+                          },
+                          child: Text(
+                            "Username"
+                          )
+                      )
+                    ],
+                  ),
+                ),
+                SizedBox(width: 12,),
+                Text(
+                  "on 99/99/3000"
+                ),
+                SizedBox(width: 12,),
+              ],
+            )
+            ]
+          )
+        )
+    );
```

Comment:
In general se pun virgule dupa majoritate parantezelelor ca sa poti da auto-format (Ctrl+Alt+L) la cod si sa nu stai sa identezi manual tot.
Daca dai autoformat fara virgule o sa se transforme codul in ceva de genul
```
 return Center(child: Card( child: Column(child: 
)));
```
which gets unreadable fast :))
Daca auto-format iti pune 2 paranteze inchise una dupa alta cel mai probabil trebuie sa pui o virgula intre ele.